instrumentation/logback/logback-appender-1.0/javaagent/src/test/groovy/LogbackTest.groovy

```diff
@@ -44,6 +44,13 @@ class LogbackTest extends AgentInstrumentationSpecification {
       waitForTraces(1)
     }
 
+    String jvmVersion = System.getProperty("java.vm.specification.version")
```

Comment:
can we use this instead? (this is the property we check for java version in the agent)
```suggestion
    String jvmVersion = System.getProperty("java.version")
```

Reply:
No. `System.getProperty("java.version")` contains the minor JVM version and the number parsing just after will fail